### PR TITLE
V2 -> V3 Migration tool

### LINKED ISSUE #3418: Initial command line Ushahidi v2 importer
Aha! Link: https://ushahiditeam.aha.io/features/PROD-270

### PR DESCRIPTION
**Pull request by rjmackay**
This is the combined branch of *all* the migration tool work so far.

Test checklist:
- Run an import
- [ ] composer test
- [ ] I certify that I ran my checklist

Fixes #3418 

Ping @ushahidi/platform

Aha! Link: https://ushahiditeam.aha.io/features/PROD-52

### CODE REVIEW

**Comment by tuxpiper** [comment]
#3771 fixes the merge conflicts

File: src/App/ImportUshahidiV2/Console/ImportUshahidiV2Command.php [hunk 1]
```diff
@@ -37,7 +40,8 @@ class ImportUshahidiV2Command extends Command
                             {database : The name of the database to import}
                             {--u|user= : The username to connect to the DB}
                             {--p|password= : The password to connect to the DB}
-                            {--host= : The database host to connect to}
+                            {--H|host= : The database host to connect to}
+                            {--P|port= : The password to connect to the DB}
```
Comment (rowasc) [inline src/App/ImportUshahidiV2/Console/ImportUshahidiV2Command.php]: --P|password?

**Comment by rowasc** [comment]
@tuxpiper maybe it's time to prepare to get this into mainstream? :grimacing:

**Comment by tuxpiper** [comment]
@Mh-Asmi @webong I have updated this branch with the latest from develop to make it mergeable again.

Let's have an async conversation in Slack about the contents and raise any concerns for merging. I'll be posting there soon.

**Comment by tuxpiper** [comment]
No concerns raised, merging to go forward with other changes

**Pull request merged**